app/subscriber/src/features/home/home-filters/styled/HomeFilters.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import { Row } from 'tno-core';
+
+export const HomeFilters = styled(Row)`
+  /* special use case for buttons */
+  @media (max-width: 500px) {
+    margin-left: 0.5rem;
+    button {
+      min-width: 4rem !important;
```

Comment:
This is a unique use case for our button on small screens, I could add it to our tno-core button; however, it may not always be desired behaviour in other cases. Also updating a few things in this PR as we speak. Will turn from draft to ready for review when done!